app/src/main/res/drawable/ic_search_48dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="48dp"
+        android:height="48dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#000"
```

Comment:
You should create a copy of this with `fillColor='#fff'` for night mode

Reply:
Oh right, unlike upload icon that was special (black also during night) here separate night icon would be better.